platform.md

```diff
@@ -1,10 +1,9 @@
 # Platform Interface Specification
 
-This document specifies the interface between a lifecycle and a platform.
+This document specifies how to process buildpack lifecycle steps to ensure compatibility
+between end-user tools.
 
-A platform orchestrates a lifecycle to make buildpack functionality available to end-users such as application developers.
-
-Examples of a platform might include:
+Examples of tools implementing Platform Interface might include:
```

Comment:
```suggestion
Examples of platforms might include:
```
Not sure if a "cloud application platform" is clearly a tool

Reply:
Well, it is a not a platform either if it doesn't implement the interface.